README.md

```diff
@@ -1,5 +1,8 @@
 # hello-world
-just a test
+
+minh muon ghi tieng viet vao day
+vi minh la nguoi viet nam
+co van de gi khong cac ban
```

Comment:
sorry but you have to write everything in English in order to help people contribute in this site.